events/2022-january-15/group_agenda.md

```diff
@@ -0,0 +1,95 @@
+#  Jupyter Accessibility Workshops with Frank Elvasky: How to build inclusive data representations
+
+## Event details
+
+Date: January 15, 2022
+
+Overview: A friendly virtual event that brings Jupyter and PyData community members together to learn from an accessibility expert. 
+
+## Takeaways
+
+Jupyter and PyData members will leave better informed and more empathetic to accessibility considerations in their work while the accessibility community will become familiar with Jupyter and its open-source challenges.
+
+## What you need to participate
+
+* Some familiarity with data/visualization skills
```

Comment:
Requested by speaker. Is there anything else we can think of/we want to call out as optional?

Reply:
Added! Good point.